globus_cli/services/transfer/endpoint/share.py

```diff
@@ -1,18 +1,18 @@
 import click
 
 from globus_cli.parsing import (
-    common_options, endpoint_create_and_update_opts, endpoint_id_arg)
+    common_options, endpoint_create_and_update_params, endpoint_id_arg)
 from globus_cli.helpers import print_json_response
 from globus_cli.services.transfer.helpers import (
     get_client, assemble_generic_doc)
 
 
-@click.command('create', help='Create a new Share, hosted on an Endpoint')
+@click.command('share', help='Create a new Share, hosted on an Endpoint')
 @common_options
-@endpoint_create_and_update_opts(create=True, shared_ep=True)
 @endpoint_id_arg(metavar='HOST_ENDPOINT_ID')
-def share_create(endpoint_id, display_name, description, organization,
-                 contact_email, contact_info, info_link, public):
+@endpoint_create_and_update_params(create=True, shared_ep=True)
+def endpoint_create_share(endpoint_id, display_name, description, organization,
+                          contact_email, contact_info, info_link, public):
```

Comment:
This one doesn't work.

```
(.venv3) ~/Repos/globus-cli$ globus transfer endpoint share bedc9334-97bd-11e6-b0b1-22000b92c261 "My Nifty Share"
TypeError: endpoint_create_share() missing 1 required positional argument: 'public'
```

Share endpoints must have `public=True` (or omitted) anyway.